tests/acceptance/FormEditorUndoAndRedoCest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Form;
+use MailPoet\Test\DataFactories\Segment;
+
+class FormEditorUndoAndRedoCest {
+  public function addNamesToAForm(\AcceptanceTester $i) {
+    $segmentFactory = new Segment();
+    $segmentName = 'Fancy List';
+    $segment = $segmentFactory->withName($segmentName)->create();
+    $formName = 'My fancy form';
+    $form = new Form();
+    $form->withName($formName)->withSegments([$segment])->create();
+    $i->wantTo('Add first and last name to the editor');
+    $i->login();
+    $i->amOnMailPoetPage('Forms');
+    $i->waitForText($formName);
+    $i->clickItemRowActionByItemName($formName, 'Edit');
+    $i->waitForElement('[data-automation-id="form_title_input"]');
+    // Add First name block
```

Comment:
Instead of using comments in the Acceptance tests, you should use `$i->wantTo()`. It has the same meaning and can be displayed in the output if using verbose output.